File: R/train/01.R
box::use(./R/lib/string[...])
box::use(./R/lib/map[...])


cypher = map.define(from = seq(1, length(letters)),
                    to = letters)

input = c(5, 24, 3, 5, 12, 12, 5, 14, 20)

(ans_train_01 = map.get(cypher, input) |> concat())

File: src/train/01.jl
# Training 01: map nums to letters

alphabet = 'a':'z'
keys = range(1, length(alphabet))
cypher = Dict(keys .=> alphabet) # <- broadcasts the dict mapper :) 

input = (5, 24, 3, 5, 12, 12, 5, 14, 20) 

# index an array out of dict
ans_train_01 = map(x -> getindex(cypher, x), input) |> join

